native/src/main/java/io/ballerina/stdlib/mime/util/EntityBodyHandler.java

```diff
@@ -456,6 +509,17 @@ public static BStream getByteStream(BObject entityObj) {
                 (ENTITY_BYTE_STREAM) : null;
     }
 
+    /**
+     * Obtains the byte stream if the content type is text/event-stream.
+     *
+     * @param entityObj Represent a ballerina entity
+     * @return A Ballerina byte stream
+     */
+    public static BStream getEventStream(BObject entityObj) {
```

Comment:
Don't we have a functionality like `setEventStream`? Since we are having separate write stream methods for byte stream and event stream. It is better to have a different setter method also. So when we set the event stream in the response we call that particular method to set the event stream in the entity body

Reply:
As discussed offline during the code review. We won't be needing the setter. Hence resolving this comment.